src/etc/inc/auth.inc

```diff
@@ -1362,29 +1362,65 @@ function ldap_backed($username, $passwd, $authcfg) {
 	return true;
 }
 
-function radius_backed($username, $passwd, $authcfg, &$attributes = array()) {
+function radius_backed($username, $password, $authcfg, &$attributes = array()) {
 	global $debug, $config;
 	$ret = false;
 
 	require_once("radius.inc");
+	require_once("CHAP.inc");
```

Comment:
CHAP.inc is not available in the tree anymore, you should require Crypt/CHAP.php here